ecosystem/typescript/sdk/src/types.ts

```diff
@@ -1,4 +1,4 @@
 // A convenience re-export from the lower level generated code
-import * as Types from "./api/data-contracts";
+import * as Types from './api/data-contracts';
```

Comment:
👍

Reply:
And also consult rest of team on these kind of changes, instead of making unilateral decisions that will affect multiple repos... it would be great to undo this change.